usagov/usagov-2021
# USAGOV-2104: USAGOV-2104 Changes to the updated Published Pages Report

Some small changes to the updated Published Pages Report


https://cm-jira.usa.gov/browse/USAGOV-2104

## Description
Altering the output of the CSV by creating a wrapper to calls to `fputcsv()`

## Type of Changes


## Testing Instructions
witch to this branch and run the `bin/static-site` script. Then check the file at `web/modules/custom/usagov_ssg_postprocessing/files/published-pages.csv` and expect to see the changes mentioned on the ticket applied.

## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
